package.json

```diff
@@ -1,92 +1,101 @@
 {
-  "name": "github-api",
-  "version": "2.4.0",
+  "name": "@mathieudutour/github-api",
+  "version": "3.2.0",
```

Comment:
it's not, I will roll back those changes. I wasn't sure you were interested in merging such a PR so I waited for that first ;)